CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 ### Fixed
 
 - [#2238](https://github.com/thanos-io/thanos/pull/2238) Ruler: Fixed Issue #2204 bug in alert queue signalling filled up queue and alerts were dropped
+- [#2231](https://github.com/thanos-io/thanos/pull/2231) Bucket Web - Sort chunks by thanos.downsample.resolution for better grouping
```

Comment:
Is this really Fixed? I think we might rather put it to `changed`?

Reply:
I think it was previously sorted properly but was changed when things migrated to block.MetaFetcher

Reply:
Yes, I think it should be in `Fixed`. The fetching of different metas happens concurrently now and I think previously that wasn't the case.

Reply:
I agree with @kfdm here. I had to recheck the code and it seems like @kfdm is correct.